i18n/index.ts

```diff
@@ -1,30 +1,53 @@
-import en from './locales/en.json'
-import ja from './locales/ja.json'
-import pt from './locales/pt.json'
-import ru from './locales/ru.json'
-import de from './locales/de.json'
-import cn from './locales/cn.json'
-import fr from './locales/fr.json'
-import tl from './locales/tl.json'
+type CustomLocale = {
+    [key in keyof typeof import('~/typedefs/gqlTypes').Locale]: string
+}
 
-const i18n = {
-    //eslint-disable-next-line
-    en_US: en,
-    //eslint-disable-next-line
-    ja_JP: ja,
-    //eslint-disable-next-line
-    pt_BR: pt,
-    //eslint-disable-next-line
-    ru_RU: ru,
-    //eslint-disable-next-line
-    de_DE: de,
-    //eslint-disable-next-line
-    zh_CN: cn,
-    //eslint-disable-next-line
-    fr_FR: fr,
-    //eslint-disable-next-line
-    tl_PH: tl
+// HACK: Browser's code language is using "-" instead of "_".
+const Locale: Partial<CustomLocale> = {
```

Comment:
can you add a comment here and on customLocale to explain what the partial and customLocale is for? I think I understand, but other devs might not.